packages/godwoken/src/client.ts

```diff
@@ -101,6 +101,19 @@ export class GodwokenClient {
     return await this.rpcCall("get_data", dataHash, toHex(blockParameter));
   }
 
+  // Don't log `invalid exit code 83` error
```

Comment:
here why leave such a comment?

Reply:
Will throw an error `invalid exit code 83` if not found this account when find script hash by eth address by registry contract, using origin `this.rpcCall` will `console.log` this error everytime when account not found